frontend/src/store/FamLoginUserState.ts

```diff
@@ -215,66 +228,93 @@ const getCachedAppRolesForDelegatedAdmin = (
             : 1;
     });
 };
+//--------- get my permissions
+const getMyFamAdminPermission = (access: FamAuthGrantDto): IMyPermission[] => {
+    const famGrant = access.grants.find(
+        (grant: FamGrantDetailDto) =>
+            grant.application.name === FAM_APPLICATION_NAME
+    );
+    if (!famGrant) return [];
```

Comment:
Hi Jota, I think we could assume famGrant should always has value. so just return
```
return [
        {
            application: famGrant!.application.description,
            env: famGrant!.application.env,
            role: 'Admin',
        },
    ];
```